server/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oo7-daemon"
```

Comment:
This needs to be added to the CI as well so that it always passes before merging any further things

Reply:
looking into this now.

Reply:
done.